app/webpacker/stylesheets/application.scss

```
@@ -12,6 +12,10 @@ $button-colour: #00823b;
 @import "warning-summary";
 @import "definition-list";
 
+.govuk-list li .govuk-body:last-child {
```

```suggestion
.govuk-list li > .govuk-body:last-child {
```

Should we only apply this to paragraphs that are direct descendants of an li element?